File: Programming Basics/04-For Loop - Exercise/salary.js
function salary(input) {

    const FACEBOOK = 150.00;
    const INSTAGRAM = 100.00;
    const REDDIT = 50.00;

    let tabs = Number(input[0]);
    let salary = Number(input[1]);

    for(let i = 2; i <= input.length; i++) {
        switch(input[i]) {
            case "Facebook":
                salary -= FACEBOOK;
                break;
            case "Instagram":
                salary -= INSTAGRAM;
                break;
            case "Reddit":
                salary -= REDDIT;
                break;
        }
        if(salary <= 0){
            console.log(`You have lost your salary.`);
            break;
        }
    }

    if(salary > 0) {
        console.log(salary);
    }
    
}

salary(["3",
"500",
"Github.com",
"Stackoverflow.com",
"softuni.bg"])

File: Programming Basics Java/src/ForLoopExercise/Salary.java
package ForLoopExercise;

import java.util.Scanner;

public class Salary {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int numberOfTabs = Integer.parseInt(scanner.nextLine());
        double salary = Double.parseDouble(scanner.nextLine());

        for (int i = 1; i <= numberOfTabs; i++) {
            String visitedSite = scanner.nextLine();

            switch (visitedSite) {
                case "Facebook":
                    salary -= 150;
                    break;
                case "Instagram":
                    salary -= 100;
                    break;
                case "Reddit":
                    salary -= 50;
            }

            if (salary <= 0) {
                break;
            }
        }

        if (salary > 0) {
            System.out.println((int)salary);
        } else {
            System.out.println("You have lost your salary.");
        }
    }
}